tests/roots/test-ext-autodoc/target/typed_vars_py35.py

```diff
@@ -0,0 +1,11 @@
+#: attr1
```

Comment:
`py35` suffix is no longer needed because Sphinx supports py35 or above.

I think `no-value` option is not related to "typed variables" at all. So no reason to add new examples. How about using `target.integer` instead?